add Interpreting the Results page to the Cookbook (#129)
RMI-PACTA/pacta.loanbook: vignettes/cookbook_interpretation.Rmd

The primary columns/variables of interest in the output are the `production` and `metric` variables. What the `production` variable represents depends on the values in all of the columns that precede it, and in particular the `metric` column. If the value in the `metric` column on the same row is `"projected"`, then the value in the `production` variable will represent the production level of your portfolio based on the ABCD data for that sector/technology/year/scenario/etc. If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `metric` column on the same row is `"corporate_economy"`, then the value in the `production` variable will represent the total production levels of the entire sector based on the ABCD for a given region, which can be used as a benchmark to compare to your portfolio's technology mix or production volume trajectory.

> For example, the output table could be filtered to view the projected coal power production values for of your portfolio with:

Author: ```suggestion
For example, the output table could be filtered to view the projected coal power production values for your portfolio with:
```

)
```

## Target Market Share

> [brief description of Target Market Share metric]

Reviewer: This should be completed.
@cjyetman presumably this, and all other content in [ ] brackets are things that you expect @jacobvjk to fill out?

For detailed information about the Target Market Share metric and how it is calculated, see the [Metrics appendix](cookbook_metrics.html#market-share-approach).

### `target_market_share()` function

The `target_market_share()` function calculates the Target Market Share metric and outputs a table including the calculated targets at the portfolio level or at the company level, depending on the options selected.

#### Explanation of Output

> The primary columns/variables of interest in the output are the `production` and `metric` variables. What the `production` variable represents depends on the values in all of the columns that precede it, and in particular the `metric` column. If the value in the `metric` column on the same row is `"projected"`, then the value in the `production` variable will represent the production level of your portfolio based on the ABCD data for that sector/technology/year/scenario/etc. If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `metric` column on the same row is `"corporate_economy"`, then the value in the `production` variable will represent [the total production levels of the entire sector adjusted by the start year production level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels].

Reviewer: I think this area would benefit from 1 code snippet illustrating this information with demo data.
Perhaps filtering and demonstrating what the dataset's `production` might look like for a single `metric`.

It doesn't need to be a long code snippet, just as it stands, it is a lot of text that is likely to be a bit confusing without a visual queue.

> The primary columns/variables of interest in the output are the `production` and `metric` variables. What the `production` variable represents depends on the values in all of the columns that precede it, and in particular the `metric` column. If the value in the `metric` column on the same row is `"projected"`, then the value in the `production` variable will represent the production level of your portfolio based on the ABCD data for that sector/technology/year/scenario/etc. If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `metric` column on the same row is `"corporate_economy"`, then the value in the `production` variable will represent [the total production levels of the entire sector adjusted by the start year production level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels].

Reviewer: Why is the last part of this sentence in square brackets?

Author: because I'm not sure if it's true/correct

Reviewer 2: I updated this section

> The primary columns/variables of interest in the output are the `production` and `metric` variables. What the `production` variable represents depends on the values in all of the columns that precede it, and in particular the `metric` column. If the value in the `metric` column on the same row is `"projected"`, then the value in the `production` variable will represent the production level of your portfolio based on the ABCD data for that sector/technology/year/scenario/etc. If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `metric` column on the same row is `"corporate_economy"`, then the value in the `production` variable will represent [the total production levels of the entire sector adjusted by the start year production level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels].

Author: @jacobvjk I think maybe you addressed this concern already in your updates/suggestions? Can you check again please?

> The primary columns/variables of interest in the output are the `production` and `metric` variables. What the `production` variable represents depends on the values in all of the columns that precede it, and in particular the `metric` column. If the value in the `metric` column on the same row is `"projected"`, then the value in the `production` variable will represent the production level of your portfolio based on the ABCD data for that sector/technology/year/scenario/etc. If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `metric` column on the same row is `"corporate_economy"`, then the value in the `production` variable will represent [the total production levels of the entire sector adjusted by the start year production level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels].

Reviewer 2: Can't make a direct suggestion here, but I will update to:
"If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. This target level of production is derived based on the ABCD production value of your portfolio in the start year follows the relative required changes of the scenario based on the market share approach for the given year.

> The primary columns/variables of interest in the output are the `production` and `metric` variables. What the `production` variable represents depends on the values in all of the columns that precede it, and in particular the `metric` column. If the value in the `metric` column on the same row is `"projected"`, then the value in the `production` variable will represent the production level of your portfolio based on the ABCD data for that sector/technology/year/scenario/etc. If the value in the `metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `production` variable will represent the target level of production for that sector/technology/year/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `metric` column on the same row is `"corporate_economy"`, then the value in the `production` variable will represent [the total production levels of the entire sector adjusted by the start year production level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels].

Reviewer 2: done from my end, if we are all happy with that

> The `technology_share` and `percentage_of_initial_production_by_scope` variables also contain values of interest, but they are derivatives of the values contained in the `production` variable. The `technology_share` variable tells the share of the row's technology within its sector, relative to the production values of other technologies in the same sector, similar to what is shown in the "technology mix" plots, relative to whatever is defined in the `metric` column in that row. The `percentage_of_initial_production_by_scope` variable shows the percent change of the production value from the prior year.

Reviewer: ```suggestion
The `technology_share` and `percentage_of_initial_production_by_scope` variables also contain values of interest, but they are derivatives of the values contained in the `production` variable. The `technology_share` variable tells the share of the row's technology within its sector, relative to the production values of other technologies in the same sector, similar to what is shown in the [technology mix plots](cookbook_interpretation.html#technoloy-mix-plot), relative to whatever is defined in the `metric` column in that row. The `percentage_of_initial_production_by_scope` variable shows the percent change of the production value from the prior year.
```

If you use the `by_company = TRUE` option of the `target_market_share()` function, most of what's said above is true except relative to the company listed in that row in the `name_abcd` column.

For detailed information about the table output of the `target_market_share()` function and its columns, see the [data dictionary](data_outputs.html#target_market_share-output).

#### Interpretation & Intuition

[2 or 3 sentences about interpretation]

[2 or 3 sentences about intuition]

### Technoloy Mix Plot

The technology mix metric focuses on technology shifts within the power, fossil fuels and automotive sectors, namely: (i) the changes in the technological processes by which outputs are produced (e.g. shift from coal-fueled to renewable-fueled power capacity), and (ii) changes in the nature of the output itself (e.g. shift from internal combustion engines to electric vehicles). This metric measures the bank’s relative exposure to the economic activities that are impacted by the transition to a low-carbon economy.

#### Explanation of Output

> [2 or 3 sentence explanation on plot output]

Reviewer: "Explanation of Output" seems to be missing? I'm actually not even sure if it's necessary, the plot is pretty self-explanatory. 

But maybe good to just briefly describe it for the sake of completeness

Author: yeah, I was unsure here.... maybe since under the top heading it more describes the "metric", here it would be good to describe how this particularly plot shows that metric?

Reviewer: Yeah exactly, I think it's a good idea to maybe just add a few quick sentences literally just explaining the plot "the top bars correspond to the portfolio and benchmark at year 2020, etc."

Author: @jacobvjk added explanation of the tech mix plot

#### Interpretation & Intuition

[2 or 3 sentences about interpretation]

[2 or 3 sentences about intuition]

```{r, message=FALSE, warning=FALSE, echo=FALSE}
library(pacta.loanbook)

market_share_targets_portfolio <-
  target_market_share(
    data = prioritize(match_name(loanbook_demo, abcd_demo)),
    abcd = abcd_demo,
    scenario = scenario_demo_2020,
    region_isos = region_isos_demo
  )

> qplot_techmix(

Author: fixed in https://github.com/RMI-PACTA/pacta.loanbook/pull/129/commits/2a193ea74e90db0cd488c466552c7c322a070181

I was generating the tmsr and sda results on the fly, now I use the included tmsr and sda demo datasets instead

  dplyr::filter(
    market_share_targets_portfolio,
    scenario_source == "demo_2020",
    sector == "power",
    region == "global",
    metric %in% c("projected", "corporate_economy", "target_sds")
  )
)

```

### Trajectory Plot

> The production volume trajectory metric aims to measure the alignment of a portfolio’s projected production volumes, based on the five-year capital plans of companies, to those given in climate scenarios. It is used for the fossil fuels, power, and automotive sectors. Changes in production volumes result either from transfer of production from one technology to another (e.g. internal combustion engines to electric vehicles) or from sheer expansion or contraction in the production coming from the technology/fuel (e.g. a company brings a new coal-fired power plant online). Projected production volumes at a 5-year horizon are considered at the individual client level at the technology level. The resulting volume trajectories are then compared with the trends set as targets in climate scenarios.

Reviewer: ```suggestion
The production volume trajectory metric aims to measure the alignment of a portfolio’s projected production volumes, based on the five-year capital expenditure plans of companies, to those given in climate scenarios. It is used for the fossil fuels, power, and automotive sectors. Changes in production volumes result either from transfer of production from one technology to another (e.g. internal combustion engines to electric vehicles) or from sheer expansion or contraction in the production coming from the technology/fuel (e.g. a company brings a new coal-fired power plant online). Projected production volumes at a 5-year horizon are considered at the individual client level at the technology level. The resulting volume trajectories are then compared with the trends set as targets in climate scenarios.
```

#### Explanation of Output

> [2 or 3 sentence explanation on plot output]

Reviewer: Again, explanation is missing.
Trajectory plot feels like it might actually need some type of explanation "falling between in a shaded regions means you are between the trajectory of scenarios X and Y, for the specific sector and technology... blah blah

Author: @jacobvjk added this

#### Interpretation & Intuition

[2 or 3 sentences about interpretation]

[2 or 3 sentences about intuition]

```{r, message=FALSE, warning=FALSE, echo=FALSE}
library(pacta.loanbook)

market_share_targets_portfolio <-
  target_market_share(
    data = prioritize(match_name(loanbook_demo, abcd_demo)),
    abcd = abcd_demo,
    scenario = scenario_demo_2020,
    region_isos = region_isos_demo
  )

qplot_trajectory(
  dplyr::filter(
    market_share_targets_portfolio,
    sector == "power",
    technology == "renewablescap",
    region == "global",
    scenario_source == "demo_2020"
  )
)
```

## Sectoral Decarbonization (SDA) Target

[brief description of SDA metric]

For detailed information about the Sectoral Decarbonization (SDA) metric and how it is calculated, see the [Metrics appendix](cookbook_metrics.html#sectoral-decarbonization-approach).

### `target_sda()` function

The `target_sda()` function calculates the Sectoral Decarbonization Target metric and outputs a table including the calculated targets at the portfolio level or at the company level, depending on the options selected.

#### Explanation of Output

> The primary column/variable of interest in the output is the `emission_factor_value` variable. What the `emission_factor_value` variable represents depends on the values in all of the other columns, and in particular the `emission_factor_metric` column. If the value in the `emission_factor_metric` column on the same row is `"projected"`, then the value in the `emission_factor_value` variable will represent the physical emission intensity level of your portfolio based on the ABCD data for that sector/year/region/scenario/etc. If the value in the `emission_factor_metric` column on the same row begins with `"target_"`, e.g. `"target_cps"`, then the value in the `emission_factor_value` variable will represent the target level of physical emission intensity for that sector/year/region/scenario/etc. based on the ABCD start year production value of your portfolio modified to be relative to the scenario data for the given year. If the value in the `emission_factor_metric` column on the same row is `"corporate_economy"`, then the value in the `emission_factor_value` variable will represent [the total physical emission intensity levels of the entire sector adjusted by the start year physical emission intensity level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels]. If the value in the `emission_factor_metric` column on the same row starts with `"adjusted_"`, then the value in the `emission_factor_value` variable will represent [?].

Reviewer: > If the value in the `emission_factor_metric` column on the same row is `"corporate_economy"`, then the value in the `emission_factor_value` variable will represent [the total physical emission intensity levels of the entire sector adjusted by the start year physical emission intensity level of your portfolio, which can be used aa a benchmark to compare to your portfolio's production levels]. If the value in the `emission_factor_metric` column on the same row starts with `"adjusted_"`, then the value in the `emission_factor_value` variable will represent [?].

"Adjusted" means that the average emission intensity of the sector has been re-baselined to the average emission intensity of the ABCD. 

So for example, if a climate scenario thinks that the average emission intensity of the steel sector is 0.5, but the ABCD has the average globally as 0.7, then it would be pretty tough to actually align to the scenario. 

Honestly I don't even know if the concept of "adjusting" makes any sense, I think this was a Klaus thing from ages ago, but that's the state of it haha

Reviewer: More information can be found here:
https://rmi-pacta.github.io/pacta.loanbook/pr-preview/pr-129/articles/cookbook_metrics.html#sectoral-decarbonization-approach

Under the "Approximation: Adjust base year scenario emission intensity" section. 

In this section, I would suggest giving a VERY brief explanation of what `adjusted` means, and then link to the Metrics documentation and that heading specifically for a more detailed breakdown.

If you use the `by_company = TRUE` option of the `target_sda()` function, most of what's said above is true except relative to the company listed in that row in the `name_abcd` column.

For detailed information about the table output of the `target_sda()` function and its columns, see the [data dictionary](data_outputs.html#target_sda-output).

#### Interpretation & Intuition

[2 or 3 sentences about interpretation]

[2 or 3 sentences about intuition]

### Emission Intensity Plot

> The emission intensity metric measures the average CO^2^ intensity of the portfolio in the steel and cement sector. This emission intensity is given as CO^2^/economic unit of output (for example, CO^2^/per ton of steel produced). This is then compared to an emission intensity reference point set by a climate scenario.

Reviewer: Hard coding "steel and cement" is likely to fall out of sync. 
Can you encode this programmatically using the sectors found in the `co2_intensity_scenario_demo`? 

It's also not a perfect solution, but we are more likely to remember updating that dataset if we add a new sector, then we are to remember updating this.

> While this is not the main metric of choice for the largest sectors tackled in this methodology, the emission intensity of the activities financed by the portfolio is nonetheless the first metric in sectors for which no clear technology pathways have been set out (namely, steel and cement). Put differently, for these sectors, no zero-carbon alternative yet exists. As such, it is not possible to use the technology mix metric or the volume production volume trajectory metric to measure alignment. However, it is still imperative to steer capital in a way that aims to decrease carbon emissions in these sectors – hence the emission intensity metric is used.

Reviewer: ```suggestion
The emission intensity of the activities financed by the portfolio is the metric of choice in sectors for which no clear technology pathways have been set out. Put differently, for these sectors, no zero-carbon alternative yet exists. As such, it is not possible to use the technology mix metric or the volume production volume trajectory metric to measure alignment. However, it is still imperative to steer capital in a way that aims to decrease carbon emissions in these sectors – hence the emission intensity metric is used.
```

Author: done in https://github.com/RMI-PACTA/pacta.loanbook/pull/129/commits/da4a94787ac71ea4173f10fa49ad7227686ff774

#### Explanation of Output

[2 or 3 sentence explanation on plot output]

#### Interpretation & Intuition

[2 or 3 sentences about interpretation]

[2 or 3 sentences about intuition]

```{r, message=FALSE, warning=FALSE, echo=FALSE}
library(pacta.loanbook)

sda_targets <-
  target_sda(
    data = prioritize(match_name(loanbook_demo, abcd_demo)),
    abcd = abcd_demo,
    co2_intensity_scenario = co2_intensity_scenario_demo,
    region_isos = region_isos_demo
  )

qplot_emission_intensity(
  dplyr::filter(
    sda_targets,

>     sector == "cement",

Reviewer: ```suggestion
    sector == "steel",
```

For whatever reason, the steel demo data seems to have more space between each line, making it a bit easier to interprate.

    abcd = abcd_demo,
    scenario = scenario_demo_2020,
    region_isos = region_isos_demo
  )

qplot_techmix(
  dplyr::filter(
    market_share_targets_portfolio,
    scenario_source == "demo_2020",
    sector == "power",
    region == "global",
    metric %in% c("projected", "corporate_economy", "target_sds")
  )
)

```

### Trajectory Plot

The production volume trajectory metric aims to measure the alignment of a portfolio’s projected production volumes, based on the five-year capital expenditure plans of companies, to those given in climate scenarios. It is used for the fossil fuels, power, and automotive sectors. Changes in production volumes result either from transfer of production from one technology to another (e.g. internal combustion engines to electric vehicles) or from sheer expansion or contraction in the production coming from the technology/fuel (e.g. a company brings a new coal-fired power plant online). Projected production volumes at a 5-year horizon are considered at the individual client level at the technology level. The resulting volume trajectories are then compared with the trends set as targets in climate scenarios.

#### Explanation of Output

> The production volume trajectory plot shows the production volume of the portfolio or company for one technology (or fuel type) at a time. The plot shows relative changes over a time fame between the start year of the analysis and five years into the future (solid black line). This projected production volume of the portfolio is compared to the production volume in the benchmark region over the same period of time (dashed black line). Finally the background of the plot is shaded in a color scale from red to yellow and green. The lines between the shaded areas represent the relative changes for the technology based on the scenario, using the market share approach. For low-carbon technologies, anything above the most ambitious scenario will be displayed in green, anything below the least ambitious scenario will be displayed in red. For high-carbon technologies, anything below the most ambitious scenario will be displayed in green, anything above the least ambitious scenario will be displayed in red. Generally, the production volume trajectory plot shows multiple scenarios from the same scenario publication, although it also works when comparing the portfolio to a single scenario (the scenario line would split the background in two shaded areas in that case).

Author: ```suggestion
The production volume trajectory plot shows the production volume of the portfolio or company for one technology (or fuel type) at a time. The plot shows relative changes over a time frame between the start year of the analysis and five years into the future (solid black line). This projected production volume of the portfolio is compared to the production volume in the benchmark region over the same period of time (dashed black line). Finally the background of the plot is shaded in a color scale from red to yellow and green. The lines between the shaded areas represent the relative changes for the technology based on the scenario, using the market share approach. For low-carbon technologies, anything above the most ambitious scenario will be displayed in green, anything below the least ambitious scenario will be displayed in red. For high-carbon technologies, anything below the most ambitious scenario will be displayed in green, anything above the least ambitious scenario will be displayed in red. Generally, the production volume trajectory plot shows multiple scenarios from the same scenario publication, although it also works when comparing the portfolio to a single scenario (the scenario line would split the background in two shaded areas in that case).
```

Another aspect about the intuition behind the more ambitious scenarios is that a higher ambition in transitioning to a low-carbon economy (scaling up low-carbon technologies and scaling down high-carbon technologies) correlates with a lower expected global heating outcome.

```{r, message=FALSE, warning=FALSE, echo=FALSE}
library(pacta.loanbook)

market_share_targets_portfolio <-
  target_market_share(
    data = prioritize(match_name(loanbook_demo, abcd_demo)),
    abcd = abcd_demo,
    scenario = scenario_demo_2020,
    region_isos = region_isos_demo
  )

qplot_trajectory(
  dplyr::filter(
    market_share_targets_portfolio,
    sector == "power",
    technology == "renewablescap",
    region == "global",
    scenario_source == "demo_2020"
  )
)
```

## Sectoral Decarbonization (SDA) Target

> The sectoral decarbonization approach (SDA) is used to calculate PACTA alignment metrics for the aviation, cement and steel sectors. The commonality of these sectors is that there are no market ready production-pathways at the technology level (or fuel type level) available yet that can be used to describe how to decarbonize the sector activities. They are currently considered "hard-to-abate" sectors. Since there are no market-ready low-carbon alternatives available in these sectors at the moment, that can be scaled up fast, the SDA approach is used to calculate sector-level metrics instead of the technology level metrics available for sectors that use the market share approach. These sector level metrics are production-based emission intensities, such as tonnes of CO~2~ per tonne of cement or tonnes of CO~2~ per tonne of steel. Regardless of how the output unit is produced, this is a comparable measure for the entire sector. The principle idea of the SDA approach is that the required carbon intensity that a sector needs to reach in the future, given assumptions around future production volumes, can be derived by dividing a sector specific carbon budget by the expected production volume. The current GHG emissions divided by the current aggregate sector production volume gives the starting point of the emission intensity. If the actors in a sector decarbonize the production in a way that follows the average emission intensity of the sector, then the scenario assumptions will be solved for in aggregate. At the portfolio/company level, the SDA approach requires every actor to decarbonize their production following their individual trajectory that starts at their current emission intensity and converges on a common emission intensity that is the same for all actors in a sector at the end of the scenario time frame (for example in 2050). This means that a company with an already lower emission intensity will not have to reduce their emission intensity as much as a company that lags the current sector average.

Author: This will no longer show aviation, but it will automatically update to whatever sectors are available in `co2_intensity_scenario_demo`

```suggestion
The sectoral decarbonization approach (SDA) is used to calculate PACTA alignment metrics for the `r cli::format_inline("{unique(co2_intensity_scenario_demo$sector)} sector{?s}")`. The commonality of these sectors is that there are no market ready production-pathways at the technology level (or fuel type level) available yet that can be used to describe how to decarbonize the sector activities. They are currently considered "hard-to-abate" sectors. Since there are no market-ready low-carbon alternatives available in these sectors at the moment, that can be scaled up fast, the SDA approach is used to calculate sector-level metrics instead of the technology level metrics available for sectors that use the market share approach. These sector level metrics are production-based emission intensities, such as tonnes of CO~2~ per tonne of cement or tonnes of CO~2~ per tonne of steel. Regardless of how the output unit is produced, this is a comparable measure for the entire sector. The principle idea of the SDA approach is that the required carbon intensity that a sector needs to reach in the future, given assumptions around future production volumes, can be derived by dividing a sector specific carbon budget by the expected production volume. The current GHG emissions divided by the current aggregate sector production volume gives the starting point of the emission intensity. If the actors in a sector decarbonize the production in a way that follows the average emission intensity of the sector, then the scenario assumptions will be solved for in aggregate. At the portfolio/company level, the SDA approach requires every actor to decarbonize their production following their individual trajectory that starts at their current emission intensity and converges on a common emission intensity that is the same for all actors in a sector at the end of the scenario time frame (for example in 2050). This means that a company with an already lower emission intensity will not have to reduce their emission intensity as much as a company that lags the current sector average.
```

Author: decided that we *should* show aviation, so closing this suggestion

> The sectoral decarbonization approach (SDA) is used to calculate PACTA alignment metrics for the aviation, cement and steel sectors. The commonality of these sectors is that there are no market ready production-pathways at the technology level (or fuel type level) available yet that can be used to describe how to decarbonize the sector activities. They are currently considered "hard-to-abate" sectors. Since there are no market-ready low-carbon alternatives available in these sectors at the moment, that can be scaled up fast, the SDA approach is used to calculate sector-level metrics instead of the technology level metrics available for sectors that use the market share approach. These sector level metrics are production-based emission intensities, such as tonnes of CO^2^ per tonne of cement or tonnes of CO^2^ per tonne of steel. Regardless of how the output unit is produced, this is a comparable measure for the entire sector. The principle idea of the SDA approach is that the required carbon intensity that a sector needs to reach in the future, given assumptions around future production volumes, can be derived by dividing a sector specific carbon budget by the expected production volume. The current GHG emissions divided by the current aggregate sector production volume gives the starting point of the emission intensity. If the actors in a sector decarbonize the production in a way that follows the average emission intensity of the sector, then the scenario assumptions will be solved for in aggregate. At the portfolio/company level, the SDA approach requires every actor to decarbonize their production following their individual trajectory that starts at their current emission intensity and converges on a common emission intensity that is the same for all actors in a sector at the end of the scenario time frame (for example in 2050). This means that a company with an already lower emission intensity will not have to reduce their emission intensity as much as a company that lags the current sector average.

Author: @jdhoffa here it mentions cement and steel *and* aviation. I was going to parameterize this like I did in the emission intensity plot section so it pulls from `co2_intensity_scenario_demo`, e.g. `r cli::format_inline("{unique(co2_intensity_scenario_demo$sector)} sector{?s}")`, but that doesn't include aviation?

Reviewer 2: the co2 scenario demo data set does not include aviation. most of the main scenarios we use do not cover all SDA sectors at the same time. At least in the past, that was a recurring phenomenon. Must have inspired us to to the same in the demo data

Author: so... should we not mention aviation here then?

Reviewer 2: We should mention it. Depending on the scenario that users compare against, they will be able to use it, so it makes sense to mention it here
